src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        Color[] colors = Color.values();
+        int randomIndex = new Random().nextInt(colors.length);
```

Comment:
make random a class-level field, instead of a local variable - it's more efficient, do we really need to recreate a random object for each getRandomColor method invocation?